docs/wiki/TestingUsage.md

```diff
@@ -36,29 +39,88 @@ Invoke-Pester -Configuration @{
 }
 ```
 
-### Handling Resource IDs or Parameters that require or contain Subscription IDs
+## Tool: Use The Test-ModuleLocally Script To Perform Pester Testing, Token Replacement and Deployment of the Module.
 
-- Scenarios where resources have dependencies on other resources, which may require to be linked using `resourceId` references. [Example](../../arm/Microsoft.Network/virtualNetworksResources/virtualNetworkPeerings/.parameters/parameters.json)
+The below example can be used on your local environment to load the 'Test-ModuleLocally.ps1' script and modify the required parameters as below to enable you to perform the tests locally before pushing to source control.
+
+```powershell
+#########[ Function Test-ModulesLocally.ps1 ]#############
+
+. 'C:\Users\user\github\user\ResourceModules\utilities\tools\Test-ModuleLocally.ps1'
```

Comment:
Maybe replace with

```
cd path/to/clonedRepo
. .\utilities\tools\Test-ModuleLocally.ps1
```

That's because the repo name (ResourceModules) may change and path to local repos may vary depending on where people are used to clone them

Reply:
Fixed